src/opendr/perception/multimodal_human_centric/rgbd_hand_gesture_learner/algorithm/data.py

```diff
@@ -65,7 +65,7 @@ def __len__(self,):
     def __getitem__(self, i):
         x, y = self.dataset.__getitem__(i)
         # change from rows x cols x channels to channels x rows x cols
-        x = np.transpose(x.numpy(), axes=(2, 0, 1))
+        x = x.convert("channels_first")
```

Comment:
Are statements like this be left on purpose? They seem to be a bit useless.

Reply:
Again, we do not specify channels order here, since we work with RGBD images.